client/js/view.js

```diff
@@ -0,0 +1,239 @@
+// This class creates the entire display and manages the virtual screens
+var View = function() {
+  // Make all the variables
+  this.init();
+  // Start animating
+  this.animate();
+};
+
+View.prototype.init = function() {
+  // Set the clock
+  this.clock = new THREE.Clock();
+
+  // Make an array to house the animation functions
+  this.animations = [];
+
+  // Create the renderer
+  this.renderer = new THREE.WebGLRenderer();
+
+  // Make a DOM element for the display to use
+  this.element = this.renderer.domElement;
+
+  // Put the entire display into the document;
+  this.container = document.getElementById('example');
```

Comment:
This can be changed to just document.body, and the matching css moved to a generic canvas selector.